easybuild/easyblocks/m/mvapich2.py

```diff
@@ -106,6 +97,11 @@ def sanity_check_step(self):
         Custom sanity check for MVAPICH2
         """
         custom_paths = {
-            'files': ['bin/%s' % x for x in ['mpiexec.mpirun_rsh']] 
+            'files': ['bin/mpiexec.mpirun_rsh'],
         }
-        super(EB_MVAPICH2, self).sanity_check_step(custom_paths=custom_paths)
+
+        # cfr. http://git.mpich.org/mpich.git/blob_plain/v3.1.1:/CHANGES
+        # MVAPICH2 2.1 is based on MPICH 3.1.4
+        use_new_libnames = LooseVersion(self.version) >= LooseVersion('2.1')
+
+        super(EB_MVAPICH2, self).sanity_check_step(custom_paths=custom_paths, use_new_libnames=use_new_libnames)
```

Comment:
The idea here was to check for `mpiexec.mpirun_rsh` in addition to all the other sanity checks from the MPICH EB.

How does it work for the sanity check? Does it return a value? Can we do something like this?
```
    custom_paths = super(EB_MVAPICH2, self).sanity_check_step(custom_paths=None, use_new_libnames=use_new_libnames)
    custom_paths.setdefault('files', []).extend(['bin/mpiexec.mpirun_rsh'])
```